src/components/organisms/Room/Participant.tsx

```diff
@@ -104,16 +104,11 @@ const Participant: React.FC<React.PropsWithChildren<ParticipantProps>> = ({
 
   const changeAudioState = useCallback(
     (enable: boolean) => {
-      if (!enable) {
-        const audioTrack = audioTracks[0];
-        if (audioTrack) {
-          audioTrack.detach();
-        }
-      } else {
-        const audioTrack = audioTracks[0];
-        if (audioTrack && audioRef.current) {
-          audioTrack.attach(audioRef.current);
-        }
+      const track = audioTracks[0];
+      const action = enable ? "attach" : "detach";
+
+      if (track && audioRef.current) {
+        track[action](audioRef.current);
```

Comment:
Maybe looks like a complicated line, still helps avoiding nesting of `if`'s which makes it clear what track is being operated on and what action will be taken on it